### PR TITLE
New machines marked as NOT_WORKING

### PR DESCRIPTION
**Pull request by clawgrip**
----------------------------------
Silverball (8.01) [jordigahan, ClawGrip]

New clones marked as NOT_WORKING
--------------------------------
Silverball (7.20) [Asure]
Silverball (6.32) [Asure]
Silverball Bulova (4.09, set 1) [Asure]
Silverball Bulova (4.09, set 2) [Asure]

### CODE REVIEW

File: src/mame/drivers/silverball.cpp [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+
+private:
+	void mem_map(address_map &map);
+	void io_map(address_map &map);
+
+	required_device<cpu_device> m_maincpu;
+};
+
+void silverball_state::mem_map(address_map &map)
+{
```
Comment (angelosa) [inline src/mame/drivers/silverball.cpp:50]: Can you please add at least conventional memory and base bios regions, i.e.:

```cplusplus
	map(0x00000000, 0x0009ffff).ram();
	map(0xfffc0000, 0xffffffff).rom().region("mb_bios", 0);
```
I don't want to compile just for this simple step, thanks in advance.

**Review by angelosa — CHANGES_REQUESTED** [review]

File: src/mame/drivers/silverball.cpp (hunk 1 above)
Comment (clawgrip) [reply src/mame/drivers/silverball.cpp:50]: If I add those two lines I got:
Intel Pentium device 'maincpu': program space is 32-bit but references 8-bit region ':mb_bios'

Can you please help? Thx!

Comment (cracyc) [reply src/mame/drivers/silverball.cpp:50]: ROM_REGION needs to be ROM_REGION32_LE

Comment (clawgrip) [reply src/mame/drivers/silverball.cpp:50]: Works fine now, thanks!

Comment (clawgrip) [reply src/mame/drivers/silverball.cpp:50]: > 
> 
> Can you please add at least conventional memory and base bios regions here, i.e.:
> 
> ```
> 	map(0x00000000, 0x0009ffff).ram();
> 	map(0xfffc0000, 0xffffffff).rom().region("mb_bios", 0);
> ```
> 
> I don't want to compile just for this simple step, thanks in advance.

@angelosa Can you please explain it? Just to learn for future PC skeleton drivers.

For the BIOS, from 0xfffc0000 to 0xffffffff it's 256K, the bigger BIOS size, it's after that? Why on the upper size? Does the BIOS always maps on the upper area? Why on 0xffffffff?

For the RAM, 640K... I'd say it's the original PC RAM limit that maps at the start, is it?

Thanks!!

**Pull request merged**